### ISSUE TITLE
SyntaxError: invalid syntax    ||    List[str] = cls.configuration()[0].get("command") 

### ISSUE THREAD

**Issue by aussetg** [sublimelsp/LSP-TexLab#10]
```python
Traceback (most recent call last):
  File "/opt/sublime_text/Lib/python33/sublime_plugin.py", line 308, in reload_plugin
    m = importlib.import_module(modulename)
  File "./python3.3/importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "/opt/sublime_text/Lib/python33/sublime_plugin.py", line 1692, in load_module
    exec(compile(source, source_path, 'exec'), mod.__dict__)
  File "/home/guillaume/.config/sublime-text/Installed Packages/LSP-TexLab.sublime-package/plugin.py", line 44
    command: List[str] = cls.configuration()[0].get("command")
           ^
SyntaxError: invalid syntax
```

All last versions

**Comment by rwols** [comment]
I fixed it in master 7c40dec

**Comment by aussetg** [comment]
> This is not in release 0.4.4: [0.4.4...master](https://github.com/sublimelsp/LSP-TexLab/compare/0.4.4...master)

Yes but you released 0.5.0, maybe by mistake :)
